samples/benchmarks/coremark/README.rst

```diff
@@ -201,6 +201,67 @@ After flashing, messages describing the benchmark state will appear in the conso
 .. note::
    |54H_engb_2_8|
 
+Alternative build configurations
+================================
+
+The default sample configuration uses the following settings for the benchmark execution:
+
+* Memory allocation using the stack memory (see the :kconfig:option:`CONFIG_COREMARK_MEMORY_METHOD_STACK` Kconfig option)
+* Single thread (see the :kconfig:option:`CONFIG_COREMARK_THREADS_NUMBER` Kconfig option)
+* Manual benchmark execution on a button press
+
+The following subsections describe alternative configurations.
+The deviation from the default configuration is described.
+Each subsection also provides a build command that demonstrates how to build the described alternative configuration.
+
+Flash and run configuration
+---------------------------
+
+The flash and run configuration changes the trigger for the benchmark execution. In this configuration, the benchmark is automatically started after the system boot.
```

Comment:
It will start on all cores if I see it correctly.

`is automatically started  on all supported cores after the system boot.`

Reply:
yes, all alternative configurations are applied to the default image and all other non-default images (e.g. `coremark_cpuppr`, `coremark_cpurad`)

Reply:
improved